source/operations/install-deploy-manage/migrate-fs-gateway.rst

```diff
@@ -0,0 +1,107 @@
+.. _minio-gateway-migration:
+
+=======================================
+Migrate from Gateway or Filesystem Mode
+=======================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Background
+----------
+
+The MinIO Gateway entered a feature freeze in July 2020.
+In February 2022, MinIO announced the `deprecation of the MinIO Gateway <https://blog.min.io/deprecation-of-the-minio-gateway/?ref=docs>`__.
+Along with the deprecation announcement, MinIO also announced that the feature would be removed in six months time.
+
+As of :minio-release:`RELEASE.2022-10-29T06-21-33Z`, the MinIO Gateway and the related filesystem mode code have been removed.
+Deployments still using the `standalone` or `filesystem` MinIO modes that upgrade to :minio-release:`RELEASE.2022-10-29T06-21-33Z` or later receive an error when attempting to start MinIO.
+
+Overview
+--------
+
+To upgrade to a :minio-release:`RELEASE.2022-10-29T06-21-33Z` or later, those who were using the `standalone` or `filesystem` deployment modes must create a new :ref:`Single-Node Single-Drive <minio-snsd>` deployment and migrate settings and content to the new deployment.
+
+This document outlines the steps required to successfully launch and migrate to a new deployment.
+
+Procedure
+---------
+
+#. Retrieve the existing deployment's **environment variables**
+
+   Use :mc-cmd:`mc admin config get` with the ``--json`` flag to retrieve a list of the environment variables defined on the existing standalone MinIO deployment.
```

Comment:
This is incorrect as this will only retrieve values that were set by mc, but would not retrieve values from /etc/default/minio or the command line that were not explicitly set via `mc admin config set`. Not sure the best path forward here but it is something like a hybrid of "if you used mc, do this, if you used env vars, do that, if you used a mix, do both"

Reply:
It should be config export and import

Reply:
👍